app/classifier/drawing-tools/root.cjsx

```diff
@@ -70,7 +70,7 @@ module.exports = React.createClass
               <TaskComponent autoFocus={i is 0} key={detailTask._key} task={detailTask} annotation={toolProps.mark.details[i]} onChange={@handleDetailsChange.bind this, i} />}
             <hr />
             <p style={textAlign: 'center'}>
-              <button autoFocus={toolProps.details[0].type in ['single', 'multiple']} type="submit" className="standard-button" disabled={not detailsAreComplete}>OK</button>
+              <button tabIndex="0" autoFocus={toolProps.details[0].type in ['single', 'multiple']} type="submit" className="standard-button" disabled={not detailsAreComplete}>OK</button>
```

Comment:
Buttons automatically have tabindex 0 so I'm not sure why this is here. The commit message says it's a fix, but not what it was fixing. Is it a workaround for my bad autofocus code on the button?

Reply:
The commit message could definitely be clearer. If the tabIndex is not indicated on the button, the dropdown task won't relinquish focus. I don't think this is an issue with the other subTasks.

Reply:
Would the thing to do here be to rebase and edit the commit message for 002e98a so that it is clearer?

Reply:
I think a comment in the code would help, so that the tabindex doesn't get deleted during a code cleanup. 

I'll make a note to have a look at the weird focus behaviour. The dropdown menus don't do that in Notes from Nature, so it might be caused by code I added to handle tabbing through drawing subtasks.

Reply:
Sounds good, I'll add a comment in the code.